jdaviz/configs/imviz/helper.py

```diff
@@ -287,10 +286,20 @@ def split_filename_with_fits_ext(filename):
     return filepath, ext, data_label
 
 
-def layer_is_image_data(layer):
+def layer_is_2d(layer):
+    # returns True for subclasses of BaseData with ndim=2, both for
+    # layers that are WCS-only as well as images containing data:
     return isinstance(layer, BaseData) and layer.ndim == 2
 
 
+def layer_is_image_data(layer):
```

Comment:
This is super confusing to me now. Maybe add docstring to explain the intention of the checks.